### PR TITLE
[MBL-16383][Teacher] Speedgrader enhancement

### PR DESCRIPTION
**Pull request by kristofnemere**
Test plan: in the ticket, can only be tested in beta environment yet. In all cases where there is a database migration, you should test that the new build works without deleting the previous build, which means that the migration was successful.

refs: MBL-16383
affects: Teacher
release note: Submission details comments are submission sensitive now.

## Checklist

- [ ] Follow-up e2e test ticket created or not needed
- [x] Run E2E test suite or not needed
- [x] Tested in dark mode
- [x] Tested in light mode
- [ ] A11y checked
- [ ] Approve from product or not needed

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

File: apps/teacher/src/main/java/com/instructure/teacher/fragments/SpeedGraderCommentsFragment.kt [hunk 1]
```diff
@@ -105,6 +105,7 @@ class SpeedGraderCommentsFragment : BaseListFragment<SubmissionCommentWrapper, S
     var mAssignmentId by LongArg()
     var mIsGroupMessage by BooleanArg()
     var mGradeAnonymously by BooleanArg()
+    var mAssignmentEnhancementsEnabled by BooleanArg()
```
Comment (tamaskozmer) [inline apps/teacher/src/main/java/com/instructure/teacher/fragments/SpeedGraderCommentsFragment.kt:108]: Why do you use hungarian notation?

**Review by tamaskozmer — APPROVED** [review]

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by hermannakos — APPROVED** [review]

**Pull request merged**